src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -0,0 +1,209 @@
+import Title from 'view/title';
+import TizenControlbar from 'view/controls/tizen/tizen-controlbar';
+import DisplayContainer from 'view/controls/display-container';
+import NextUpToolTip from 'view/controls/nextuptooltip';
+import SettingsMenu from 'view/controls/components/menu/settings-menu.js';
+import { addClass } from 'utils/dom';
+import { STATE_PLAYING, STATE_PAUSED } from 'events/events';
+import type ViewModel from 'view/view-model';
+import type { PlayerAPI } from 'types/generic.type';
+
+const Controls = require('../controls').default;
+
+const ACTIVE_TIMEOUT = 5000;
+
+const reasonInteraction = () => {
+    return { reason: 'interaction' };
+};
+
+class TizenControls extends Controls {
+    context: HTMLDocument;
+    playerContainer: HTMLElement;
+    div: HTMLElement | null;
+    backdrop: HTMLElement | null;
+    pauseDisplay: HTMLElement | null;
+    displayContainer: DisplayContainer | null;
+    controlbar: TizenControlbar | null;
+    settingsMenu: SettingsMenu | null;
+    showing: boolean;
+    keydownCallback: ((evt: KeyboardEvent) => void) | null;
+    addControls: any;
```

Comment:
i assume these would get re-typed when controls is typed? or maybe removed?

Reply:
These should be removed when controls is typed since they are methods on controls/events. They were throwing a type error when called.